p2pserver/p2pserver.go

```diff
@@ -468,7 +469,7 @@ func (this *P2PServer) timeout() {
 			t := p.GetContactTime()
 			if t.Before(time.Now().Add(-1 * time.Second *
 				time.Duration(periodTime) * common.KEEPALIVE_TIMEOUT)) {
-				log.Warn("Keep alive timeout!!!")
+				log.Warn("keep alive timeout!!!")
```

Comment:
keep alive timeout with who? from what time?

Reply:
add more info in latest commit